src/features/Managers/Popups/EditAccessPopup/EditAccessPopup.tsx

```diff
@@ -21,57 +22,112 @@ export const EditAccessPopup = ({
   const { t } = useTranslation('app');
   const { firstName, lastName, email } = user;
   const [applets, setApplets] = useState<AppletType[]>(mockedApplets);
+  const [appletsWithoutRespondents, setAppletsWithoutRespondents] = useState<string[]>([]);
+  const [editAccessSuccessPopupVisible, setEditAccessSuccessPopupVisible] = useState(false);
 
-  const updateRolesHandler = (id: string, callback: (roles: Role[]) => Role[]) => {
+  const getAppletsWithoutRespondents = () =>
+    applets.reduce((acc: string[], el) => {
+      if (
+        el.roles.some((role) => role.label === Roles.reviewer) &&
+        !el?.selectedRespondents?.length
+      ) {
+        acc.push(el.title);
+      }
+
+      return acc;
+    }, []);
+
+  const updateAppletHandler = (
+    id: string,
+    callback: (roles: Role[]) => Role[],
+    respondents?: string[] | null,
+  ) => {
     const updatedApplets = applets.map((applet) =>
       applet.id === id
         ? {
             ...applet,
             roles: callback(applet.roles),
+            selectedRespondents: respondents || applet.selectedRespondents,
           }
         : { ...applet },
     );
     setApplets(updatedApplets);
   };
 
-  const handleRemove = (id: string, label: Roles) => {
-    updateRolesHandler(id, (roles) => roles.filter((role) => role.label !== label));
+  const handleRemoveRole = (id: string, label: Roles) => {
```

Comment:
I think, you can use handleRemoveRole, handleAddRole, handleAddSelectedRespondents functions without curly braces.